### DISCUSSION TITLE
Is there a way to get the space app hostname from cli or API?

### DISCUSSION THREAD

**Discussion by begoon** [deta/space-docs#22] [General]
Is there a way to get the space app hostname from cli or API?

The value of the `DETA_SPACE_APP_HOSTNAME` variable is needed (like `abcdef-0-u12345678.deta.app`)

Previously, in `deta.sh`, this value was exposed in the `.deta` folder and printed by `deta` CLI.

After `space push`, it is convenient to call the endpoint programmatically, but without hardcoding the URL into scripts.

**Comment by abdelhai** [comment]
i think i replied on discord. not yet, good idea and I took note!
